exporters/inmemory/src/test/java/io/opentelemetry/exporters/inmemory/InMemoryTracingTest.java

```diff
@@ -33,24 +34,20 @@
 @RunWith(JUnit4.class)
 public class InMemoryTracingTest {
   private final InMemoryTracing tracing = new InMemoryTracing();
+  private final Tracer tracer = tracing.getTracerFactory().get("InMemoryTracing");
```

Comment:
I don't imagine users calling `tracing.getTracerFactory()` all the time, even when they are testing a simple scenario that doesn't require this capability at all. Maybe we can have a `getTracer()` that returns the default `Tracer`? Or even `getDefaultTracer()`.

Reply:
They need to because with the named tracer every instrumentation should accept a TracerFactory not a Tracer. This is the change that was added with the named tracer concept.

In general you should think that every instrumentation plugin will now accept a factory during initialization not a Tracer, and they will create their own named Tracer.

Reply:
Also we can see if the get default is needed later, but so far I don't see a reason to have that.

Reply:
@carlosalberto please let me know if you are ok to move forward with this.